---
title: Research
keywords: midway2
last_updated: March 22, 2023
tags: [midway]
summary: "This is an introduction to research in Pab Lab."
sidebar: sidebar_1
permalink: lab_research.html
folder: mydoc
---

{% include image.html file="Prof_Tao_Pan.jpg" alt="Prof. Tao Pan" caption="Prof. Tao Pan" %}

## Research Summary
Our research focuses on 
(i) functional genomics and biology of tRNA including microbiomes and
(ii) epitranscriptomics including microbiome-host interactions.

**tRNA biology:** Translational regulation relies on the dynamic properties of tRNA that constantly change to facilitate response and adaptation to new environments and to control gene expression. We developed high throughput sequencing technologies that measure tRNA abundance, charging and modifications in one single sequencing library. We are investigating the roles of tRNA in translational control and extra-translational functions in mammalian cells.

**Microbiome:** We also developed tRNA-seq as another approach for microbiome characterization. Standard microbiome characterizations include 16S-seq or shotgun metagenomics. Although powerful, these DNA-based methods do not directly report the microbiome activity such as dynamic gene expression which requires the studies of RNA in the microbiome. Our microbiome tRNA-seq results show extensive variations of tRNA abundance and modification patterns in microbiomes from different sources. We also show that tRNA modification dynamics in the microbiome correlates with tuning the expression of specific microbial proteins, indicating that tRNA-seq can provide new insights in microbiome biology. We are further developing this approach to explore the potentials of tRNA-seq to study microbiomes from humans and from the oceans.

**Epitranscriptomics:** Over 100 types of post-transcriptional RNA modifications have been identified in thousands of sites in the transcriptome. They include methylation of bases and the ribose backbone, rotation and reduction of uridine, base deamination, addition of ring structures and carbohydrate moieties, and so on. mRNA modifications are involved in cell differentiation, proliferation, and many other cellular functions and human diseases. Some mRNA modifications can also be removed by cellular enzymes, resulting in the dynamic regulation of their functions. We are investigating the function and mechanisms of mRNA modifications such as N6-methyladenosine (m6A) in the regulation of gene expression. For example, we discovered that m6A modification can alter the local mRNA structure to regulate binding of mRNA binding proteins transcriptome-wide (m6A switch), resulting in changes in mRNA abundance and alternative splicing.

**Microbiome-host interactions through epitranscriptomics:** We are working on elucidating the function of mammalian host mRNA and tRNA modifications in response to the gut microbiome. We found that microbiome reprograms the host m6A modifications transcriptome-wide in a tissue-dependent manner, suggesting that this dynamic epitranscriptomic mark is used in yet unknown ways in microbiome response. We also found that a microbiome dependent, host tRNA modification alters the cellular small RNA pool, suggesting yet another pathway of microbiome response through RNA modifications.







{% include links.html %}